yarn.lock

```diff
@@ -10063,6 +10117,10 @@ prr@~1.0.1:
   resolved "https://registry.yarnpkg.com/prr/-/prr-1.0.1.tgz#d3fc114ba06995a45ec6893f484ceb1d78f5f476"
   integrity sha1-0/wRS6BplaRexok/SEzrHXj19HY=
 
+"pseudo-worker@git+https://github.com/pwnsdx/pseudo-worker.git#c8bcc87e3730b0d6593d9ef5bc60df0860d84643":
```

Comment:
I am a bit concerned about this. Can we create a package from this?

Reply:
@pwnsdx Can you host `pseudo-worker` on npm?

Reply:
@ffflorian Discussed it with @pwnsdx & @Yserz. We will wait for [this](https://github.com/nolanlawson/pseudo-worker/pull/19) to be merged and then exchange "pwnsdx/pseudo-worker" with the original library where it was forked from.

Reply:
@pwnsdx & @ffflorian We can now take the official "pseudo-worker": https://github.com/nolanlawson/pseudo-worker/pull/19#issuecomment-555078828 🎉